modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -193,3 +199,170 @@ extension RequestBuilder {
         return deferred.promise
     }
 }{{/usePromiseKit}}
+
+public enum JSON: Codable, Hashable {
```

Comment:
@aymanbagabas I have been searching for other implementations of this, to see if there is one that looks the more complete, but I have found so many of them, and none seems to be a clear choice to me. 
Here are some examples:
https://github.com/owensd/json-swift
https://github.com/Weebly/JSONValue
https://github.com/rexmas/JSONValue
https://github.com/VeryLoveLoli/JSONValue
https://gist.github.com/klundberg/419de78d1b2f7a46c11b40b220ec5cd9
https://gist.github.com/rnapier/d91175be2f737aca7ad476759948666e
https://github.com/swagger-api/swagger-codegen/commit/5ef457419c0a91968a07409264c79c8c4fdc70ea

Another option that we could look at would be AnyCodable that seems to be quite well accepted.
https://github.com/Flight-School/AnyCodable

What do you think of this?

Reply:
I really like [IkigaJSON](https://github.com/Joannis/IkigaJSON), it also supports SwiftNIO and built on top of Foundation. But [AnyCodable](https://github.com/Flight-School/AnyCodable) fits exactly our use-case to replace `Any` with a `Codable` type.

Reply:
Although I like the `JSON` enum, I would prefer something more broadly accepted as `AnyCodable`, and `AnyCodable` is simple

Reply:
I'll replace `JSON` with `AnyCodable`.